qa-tester/assets/src/js/admin-bar.js

```diff
@@ -0,0 +1,218 @@
+/**
+ * WordPress dependencies
+ */
+import React, { Component, render } from '@wordpress/element';
+import domReady from '@wordpress/dom-ready';
+import apiFetch from '@wordpress/api-fetch';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import '../css/admin-bar.css';
+
+class AdminBar extends Component {
+	constructor( props ) {
+		super( props );
+
+		const buildOptions = [
+			{
+				label: __( 'Latest release', 'amp-qa-tester' ),
+				value: 'release',
+			},
+			{
+				label: __( 'Develop branch', 'amp-qa-tester' ),
+				value: 'develop',
+			},
```

Comment:
We also need to allow selecting the `1.5` branch or other release branches after `1.5`.

Reply:
Addressed in https://github.com/ampproject/amp-wp/pull/4770/commits/f0f1699284591ba4a4f5d7eb5e01c29480f3b18c.